app/domain/authentication/authn_oidc/authenticator.rb

```diff
@@ -85,6 +85,10 @@ def subject_err_msg
         "User info subject [#{user_info.sub}] and id token subject " +
          "[#{id_token_subject}] are not equal"
       end
+
+      def is_invalid_token_error? err
+        err.class <= OpenIDConnect::ResponseObject::IdToken::InvalidToken
+      end
```

Comment:
per above comment, this can be removed.

Reply:
i don't follow

Reply:
I removed it as this method is redunant by rescuing the specific error. Not sure why this is relevant to the above comment